### ISSUE TITLE
Deploy on openshift

### ISSUE THREAD

**Issue by Pablo2m** [waartaa/waartaa#28]
Hello:
Has anyone succeeded in installing waartaa on OpenShift?, Looks like a good place to run an instance

**Comment by rtnpro** [comment]
@Pablo2m 

No, I haven't tried to deploy waartaa on OpenShift yet. However, I did on nodejitsu.com (PAAS for node js apps). Initially it was running fine, but when the server load increased, the server was restarting frequently. Finally, I moved to VPS.

Please feel free to deploy waartaa on OpenShift. We can document the deployment process and even write scripts to automate it :)

**Comment by Pablo2m** [comment]
Well I started to investigate, since you can not install meteor on OpenShift  think I'll have to use this

https://github.com/onmodulus/demeteorizer

**Comment by rtnpro** [comment]
@Pablo2m

Yes! you're right. These PAAS platforms don't understand meteor yet. I use
demeteorizer myself too for deployment of https://www.waartaa.com :)

On Fri, Jan 17, 2014 at 11:10 PM, Pablo2m notifications@github.com wrote:

> Well I started to investigate, since you can not install meteor on
> OpenShift think I'll have to use this
> 
> https://github.com/onmodulus/demeteorizer
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/waartaa/waartaa/issues/28#issuecomment-32627406
> .

## 

Ratnadeep Debnath,
Software Engineer, www.goibibo.com
GPG Fingerprint: 033C 8041 A0E9 CDBA 2E02  B785 2119 5486 F245 DFD6

**Comment by rtnpro** [comment]
@praveenkumar, can you look into it?

**Comment by praveenkumar** [comment]
@rtnpro Lets have a meeting about deployment process of waartaa then will start start deployment scripts.
